core/src/main/kotlin/com/github/quillraven/darkmatter/ui/skin.kt

```diff
@@ -45,6 +50,17 @@ fun createSkin(assets: AssetStorage) {
         createLabelStyles(bigFont, defaultFont)
         createImageButtonStyles(skin)
         createTextButtonStyles(defaultFont, skin)
+        createWindowStyles(skin, defaultFont)
+    }
+}
+
+private fun @SkinDsl Skin.createWindowStyles(
```

Comment:
@SkinDsl annotation is unnecessary. Unless you're adding new methods that expand the DSL, the annotation has no effect.

Reply:
okay, Intellij added that when I was using refactor -> extract method. But if it has no effect, I will remove it

Reply:
I mean, it could have some effect, especially if there were any lambda parameters. I think case I don't think it will prevent from any bugs that the current DSL doesn't already protects against.